B827EB61FA02.json

```diff
@@ -0,0 +1,17 @@
+{
+  "gateway_conf": {
+    "gateway_ID": "B827EB61FA02",
+    "servers": [
+      {
+        "server_address": "brazil.thethings.network",
```

Comment:
I don't think that's a valid server address, it's not listed on the website at least, are you sure it is correct? https://www.thethingsnetwork.org/docs/gateways/packet-forwarder/semtech-udp.html#router-addresses